### PR TITLE
Application system should use new identity-server

### PR DESCRIPTION
**Pull request by dabbeg**
## What

Add identity-server to env and switch to new stable identity-server

## Why

To get the application system working on dev

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by darrikonn — APPROVED** [review]

**Review by arnifreyrs — APPROVED** [review]
@magnearun had already started work on this in a bigger PR, adding the new authguards to the backend, but this is still a welcome addition

File: apps/application-system/form/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 export default {
   production: true,
   baseApiUrl: process.env.API_URL,
+  identityServer: {
```
Comment (darrikonn) [inline apps/application-system/form/src/environments/environment.prod.ts:4]: Should the front-end handle the redirection?

Comment (dabbeg) [reply apps/application-system/form/src/environments/environment.prod.ts:4]: Hmm I am not sure, I don't see any reason why it should not do so.
If you can think of a reason we should take this up in the `audkenning_og_heimild` slack channel as there are more projects configured this way.

**Comment by dabbeg** [comment]
> @magnearun had already started work on this in a bigger PR, adding the new authguards to the backend, but this is still a welcome addition

Awesome! Good to know, I only wanted to make this work on dev. I will not make any more changes to this auth stuff :slightly_smiling_face:

**Pull request merged**